src/heads-up/components/properties-item.vue

```diff
@@ -5,7 +5,10 @@
         <slot />
       </span>
 
-      <app-tooltip class="properties-item__tooltip">
+      <app-tooltip
+        class="properties-item__tooltip"
+        :placement="'top-start'"
```

Comment:
Please use a static string here, so `placement="top-start"`. This way, it is overly verbose